lang/terraform/terraform.py

```diff
@@ -0,0 +1,156 @@
+from talon import Context, Module, actions, settings
+
+ctx = Context()
+mod = Module()
+ctx.matches = r"""
+mode: user.terraform
+mode: user.auto_lang
+and code.language: terraform
+"""
+
+types = {
+    "string": "string",
+    "number": "number",
+    "bool": "bool",
+    "list": "list",
+    "map": "map",
+    "null": "null"
+}
+
+ctx.lists["user.code_type"] = types
+
+common_properties = {
+    "name": "name",
+    "type": "type",
+    "description": "description",
+    "default": "default",
+    "for each": "for_each",
+    "count": "count",
+    "prevent destroy": "prevent_destroy",
+    "nullable": "nullable",
+    "sensitive": "sensitive",
+    "depends on": "depends_on",
+    "provider": "provider",
+    "source": "source",
+}
+
+mod.list("terraform_common_properties", desc="Terraform Modifiers")
```

Comment:
List names should be singular

Reply:
Thank you for the heads up. Fixed this with an amended commit.
FYI: I copied this from the java bindings, where plural is (still/erroneously?) used.